email_login/src/components/CheckboxItem.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './checkboxitem.scss';
+
+const CheckboxItem = props => (
```

Comment:
Could you rename this component? Please :)
It seems a default checkbox instead a Remove device checkbox
It's little confusing :$